modules/cudacodec/src/video_source.hpp

```diff
@@ -57,7 +57,7 @@ class VideoSource
 
     virtual FormatInfo format() const = 0;
     virtual void updateFormat(const FormatInfo& videoFormat) = 0;
-    virtual bool get(const int propertyId, double& propertyVal) const { return false; }
```

Comment:
It was for `CuvidVideoSource` but I agree it will be much cleaner to move this to cuvid_video_source.h especially as `CUvideosource` is marked for removal in the next version of the Nvidia Video Codec SDK.